src/express-server/index.ts

```diff
@@ -1,13 +1,19 @@
 import​​ app  ​from​​​ "./app"​;
 import { db } from "./config/db";
 import defaultConfig from "./config/defaultConfig.json";
+import { mongodb } from "./config/mongodb";
 
 const​​ appPort = process.env.PORT || defaultConfig.defaultPort​;
 
```

Comment:
I've got a proposition - introduce new CLI param like `--db=sql/mongo`, so when I can provide it like `npm start -- --db=sql` and make Mongo default